src/test/resources/org/openmrs/uitestframework/test-cypress.properties

```diff
@@ -0,0 +1 @@
+webapp.url=https://openmrs-spa.org/openmrs/spa
```

Comment:
Is this the baseurl that are we going to use? 

Will it be suitable for the test that has "create" operations? (ex: patient registration)

Reply:
yeah, that should be the base-url

Reply:
I thought that we were going to be running tests on isolated test servers.

Reply:
a test instance for microfrontends should be started in the github actions and waited for before running tests whose url should used here insyead

Reply:
Great. I think that @jayasanka-sack will need some help making that happen, and that we probably shouldn't block this PR on it.